### DISCUSSION TITLE
Extension recommendation like in VS Code

### DISCUSSION THREAD

**Discussion by brainwo** [neoclide/coc.nvim#4321] [Ideas]
I'm not sure if something like this already been implemented but it might be a good idea to have an extension recommendation option like in VS Code `.vscode/extensions.json` where you can specify recommended extensions on working directory then have a popup to tell you to install those extensions. This option will scan through installed extensions and recommend not installed extensions.

The idea is to put it in `.vim/coc-settings.json` like example below:
```json
{
   "coc.recommendations": [ "coc-html", "coc-json", "coc-clangd" ]
}
```

This will helps a lot in collaborative project where people don't always have the same setup.

**Comment by chemzqm** [comment]
VSCode have file .vscode/extensions.json which json like:
```json
{
	// See http://go.microsoft.com/fwlink/?LinkId=827846
	// for the documentation about the extensions.json format
	"recommendations": [
		"dbaeumer.vscode-eslint"
	]
}
```
we could support something similar.
